org.gnome.OTPClient.json

```diff
@@ -0,0 +1,83 @@
+{
+  "app-id": "org.gnome.OTPClient",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.24",
+  "sdk": "org.gnome.Sdk",
+  "command": "otpclient",
+  "rename-desktop-file": "otpclient.desktop",
+  "finish-args": [
+    "--device=all",
+    "--filesystem=home",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland"
+  ],
+  "modules": [
+    {
+      "name": "libzip",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://libzip.org/download/libzip-1.3.2.tar.xz",
+          "sha256": "6277845010dbc20e281a77e637c97765c1323d67df4d456fd942f525ea86e185"
+        }
+      ]
+    },
+    {
+      "name": "libgpg-error",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.27.tar.bz2",
+          "sha256": "4f93aac6fecb7da2b92871bb9ee33032be6a87b174f54abf8ddf0911a22d29d2"
+        }
+      ]
+    },
+    {
+      "name": "libgcrypt",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://www.gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.8.2.tar.bz2",
+          "sha256": "c8064cae7558144b13ef0eb87093412380efa16c4ee30ad12ecb54886a524c07"
+        }
+      ]
+    },
+    {
+      "name": "libbaseencode",
+      "no-autogen": true,
+      "cmake": true,
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/paolostivanin/libbaseencode/archive/v1.0.4.tar.gz",
+          "sha256": "1ed0a3dd29a0ed70c7113f1b1b7d17f72c65454946a9fb915edf5050f8e0c65d"
+        }
+      ]
+    },
+    {
+      "name": "libcotp",
+      "no-autogen": true,
+      "cmake": true,
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/paolostivanin/libcotp/archive/v1.0.11.tar.gz",
+          "sha256": "fd01f46a81a1f2811c932eab336276d982369401ae405e36034f1aa859b04a74"
+        }
+      ]
+    },
+    {
+      "name": "OTPClient",
+      "no-autogen": true,
+      "cmake": true,
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/paolostivanin/OTPClient.git",
+          "branch": "master"
```

Comment:
Please use a tag or commit here so the build is repeatable

Reply:
yep, right! I forgot to update the file after the release was tagged